src/resources/index.html

```diff
@@ -0,0 +1,169 @@
+          <span class="w-full text-center">
+            <a href="/resources/" class="link uppercase text-bright_blue border-bright_blue">resources</a>
+          </span>
+          <span class="w-full text-right">
+            <a href="/community/" class="link uppercase">community</a>
+          </span>
+        </nav>
+      </header>
+
+      <section class="w-full">
+        <div class="mb-8">
+          <h2 class="text-xl mb-3 text-center sm:text-left uppercase">Resources</h2>
+          <p class="text-justify">
+            Here are collected miscellaneous Srcery resources. In addition to
+            themes Srcery has some tools and packages to assist in creating
+            new themes, or things that didn't fit anywhere else.
+          </p>
+        </div>
+        <div class="mb-8">
+          <h3 class="mb-3 text-center sm:text-left uppercase">Palette</h3>
+          <p class="mb-4" x-show="version">
+            <span class="">Latest version:</span>
+            <a
+              href="https://www.npmjs.com/package/@srcery-colors/srcery-palette"
+              class="link" x-text="version">
+            </a>
+          </p>
+          <p class="mb-4">
+            In an attempt to create a single source of truth concerning the
+            palette used in various themes, an npm package was created that
+            contains a JSON with all color information.
+          </p>
+          <p class="mb-4">
+            It isn't used in every project, but every change to the palette will
+            happen there first, and produce a new version of this package going
+            forward.
+          </p>
+          <p class="mb-4">Install using this command or check out the links below:</p>
+
+          <code class="code">
+            npm i @srcery-colors/srcery-palette
+          </code>
+          <ul class="">
+            <li class="mb-2 flex items-center">
+              <div class="list-dot"></div>
+              <a target="_blank" class="link flex items-center" href="https://github.com/srcery-colors/srcery-palette">
+                <div class="svg mr-2 w-5" x-html="icons.github"></div>
+                Github repository
+              </a>
+            </li>
+            <li class="flex items-center">
+              <div class="list-dot"></div>
+              <a target="_blank" class="flex items-center link" href="https://www.npmjs.com/package/@srcery-colors/srcery-palette">
+                <div class="svg mr-2 w-8" x-html="icons.npm"></div>
+                <div>
+                  <span class="mr-1">npm package</span>
+                  <span x-show="version">(<strong x-show="version" x-text="version"></strong>)</span>
+                </div>
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="mb-8">
+          <h2 class="text-xl mb-4 text-center sm:text-left uppercase">Web assets</h2>
+          <p class="mb-4">Aside from the palette itself, here are some useful web assets</p>
```

Comment:
```suggestion
          <p class="mb-4">Aside from the palette itself, here are some useful web assets.</p>
```

As below, not sure if this is correct, but the reader is being pointed to a list or something.  Thoughts?